langchain/src/chat_models/openai.ts

```diff
@@ -323,6 +314,14 @@ export class ChatOpenAI
     this.clientConfig = {
       apiKey: this.openAIApiKey,
       organization: this.organization,
+      baseURL: configuration?.basePath ?? fields?.configuration?.basePath,
+      dangerouslyAllowBrowser: true,
+      defaultHeaders:
```

Comment:
Am planning to just support previously passed `basePath`, `headers`, and `params` but could shim more things in here.